### PR TITLE
feat(service) : Add letter service

### PR DESCRIPTION
**Pull request by Aril-Man**

### CODE REVIEW

File: app/Models/LetterIn.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+class LetterIn extends Model
+{
+    use HasFactory;
+
+    // sifat surat
+    const Rahasia = 'Rahasia';
+    const Penting = 'Penting';
+    const Segera = 'Segera';
+    const Biasa = 'Biasa';
+}
```
Comment (Saekyo) [inline app/Models/LetterIn.php:19]: tambah incrementingnya `public $incrementing = false;`

File: app/Models/LetterOut.php [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+class LetterOut extends Model
+{
+    use HasFactory;
+
+    // sifat surat
+    const Rahasia = 'Rahasia';
+    const Penting = 'Penting';
+    const Segera = 'Segera';
+    const Biasa = 'Biasa';
+}
```
Comment (Saekyo) [inline app/Models/LetterOut.php:19]: tambah incrementingnya `public $incrementing = false;`

Comment (Aril-Man) [reply app/Models/LetterOut.php:19]: OKee okee

**Comment by Saekyo** [comment]
LGTM

**Pull request closed without merging**